kusto/ingest/internal/resources/resources.go

```diff
@@ -17,7 +17,7 @@ import (
 
 // mgmter is a private interface that allows us to write hermetic tests against the kusto.Client.Mgmt() method.
 type mgmter interface {
```

Comment:
mgmter?
Not very readable.
I get 'mgmt', what does 'er' stand for?

Reply:
Also, if your sole purpose if testing, why not use the below Query Client.

Reply:
That's not new code of mine, Mgmter was before.

It's one of those annoying go convensions - if an interface has one method called X, the interface should be called Xer.

I could switch it to QueryClient I guess, but then I have to also change the tests that provide it a mgmter mock

Reply:
Resolved offline